app/views/admin/protocol_drugs/index.html.erb

```diff
@@ -37,7 +37,7 @@
 <br>
 
 <% if current_admin.permissions_v2_enabled? %>
-  <% if current_admin.accessible_organizations(:manage).any? %>
+  <% if current_admin.accessible_organizations(:manage).any? || current_admin.power_user? %>
```

Comment:
Shouldn't `accessible_organizations` return a non-empty set of orgs for power users?

Reply:
Yes, with one edge case: when no organizations exist in the system yet. Adding this to UI checks so the application is usable at initial setup.

Reply:
Ah the blank slate problem. Neat. 👍